openstack/identity/v3/projects/results.go

```diff
@@ -14,6 +14,16 @@ type GetResult struct {
 	projectResult
 }
 
+// CreateResult temporarily contains the reponse from the Create call.
+type CreateResult struct {
+	projectResult
+}
+
+// createErr quickly wraps an error in a CreateResult.
+func createErr(err error) CreateResult {
```

Comment:
I assumed it was part of how errors are formed somehow, but that was just an assumption. It exists in the other identity packages, so I copied it here.